activity_calendar/static/css/fullcalendar-customisation.css

```diff
@@ -42,3 +42,19 @@
     white-space: pre-line;
 }
 
+/*
+SVG Icon colours
+Generated using from hexcodes using: https://codepen.io/sosuke/pen/Pjoqqp
```

Comment:
nit: which hexcodes were used to start with?

Reply:
I can't really remember, but they were roughly based on the bootstrap alert text colours (but do not seem to precisely match)